### PR TITLE
Release/Travel Study minor updates

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- New release with the few minor updates
- Include the support for storyblok link "email" option, T/S SEO, alert fixups, misc request from Kara for T/S

# Review By (Date)
- 11/19 if possible since this would help with content entry by providing a more accurate preview

# Criticality
- 6/10

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at the preview site and check that it's behaving as expected
2. Take a quick look at the code and check that things look right

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/page-types/TripPage/TripPage.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useRef, useMemo } from 'react';
 import { dcnb } from 'cnbuilder';
 import { useReactToPrint } from 'react-to-print';
 import SbEditable from 'storyblok-react';
-import { Alert, Container } from 'decanter-react';
+import { Alert } from 'decanter-react';
```
Comment (sherakama) [inline src/components/page-types/TripPage/TripPage.jsx:5]: 🤔  We're not decoupled yet? This would be good to get on sooner than later.

**Review by sherakama — APPROVED** [review]
@yvonnetangsu sorry for not getting to this sooner for you but this looks good. The code climate errors on duplication are acceptable.

File: src/components/page-types/TripPage/TripPage.jsx (hunk 1 above)
Comment (yvonnetangsu) [reply src/components/page-types/TripPage/TripPage.jsx:5]: Not fully. I did part 1. Wanted to do a little bit more than just copy directly from Decanter React. I'll add a new ticket for the remaining components for this next sprint 😃

Comment (sherakama) [reply src/components/page-types/TripPage/TripPage.jsx:5]: For the best. Thanks for the update.

**Comment by yvonnetangsu** [comment]
@mattanglin @sherakama I'm going to go ahead and merge this in. No schema change with this release so nothing needs to be done on the Storyblok side.

After this is merged, I'll prep another small release with the masthead rendering and the itinerary hotel schema change.

**Pull request merged**